stories/components/TopBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Col from 'react-bootstrap/lib/Col';
+import { storiesOf } from '@storybook/react';
+import StoryItem from '../styleguide/StoryItem';
+import TopBar from '../../lib/TopBar';
+import Button from '../../lib/Button';
+
+storiesOf('Components', module).add('TopBar', () => (
+  <div>
+    <StoryItem
+      title="TopBar Wrapper"
+      description="TopBar is the top bar used throughout the whole site."
+    >
+      <TopBar>
```

Comment:
I think we should add more stories so we can see some of the topbars that I know we need to support;

- left, center & right
- left, right (no centre)

And maybe show some of the contents we support in this component e.g. `Avatar`.